webapp/src/pages/success/index.tsx

```diff
@@ -0,0 +1,52 @@
+/** @jsx jsx */
+import { jsx, Button } from 'theme-ui'
+import Link from 'next/link'
+import { useRouter } from 'next/router'
+
+import * as styles from './Success.styles'
+import Layout from '../../components/Layout/Layout'
+import HeaderLogo from '../../components/HeaderLogo/HeaderLogo';
+import InfoPage from '../../components/InfoPage/InfoPage';
+
+const Success = () => {
+  const router = useRouter()
+  const { playlist_id: playlistId } = router.query
+
+  return (
+    <Layout>
+      <InfoPage>
+        <HeaderLogo />
+
+        <p>
+          Success! Your playlist has been saved to your account.
+        </p>
+
+        <Link href={`https://open.spotify.com/playlist/${playlistId}`}>
+          <a target="_blank">
+            <Button
+              sx={styles.searchBtn}
+            >
+              View in Spotify
+            </Button>
+          </a>
+        </Link>
+
+        <p>
+          Or, listen to it directly here:
+        </p>
+
+        <iframe
+          sx={styles.playlist}
+          src={`https://open.spotify.com/embed/playlist/${playlistId}`}
```

Comment:
This URL works, but for some reason about 50% of the time I'd load the page I would get a `Bad Request` message in the iframe. The other 50% was peachy keen. Let's merge for now and if the problem persists when we can troubleshoot then.